packages/editor/src/components/panel-color-settings/test/__snapshots__/index.js.snap

```diff
@@ -87,7 +87,7 @@ exports[`PanelColorSettings should render a color panel if at least one setting
 `;
 
 exports[`PanelColorSettings should render a color panel if at least one setting specifies some colors to choose 2`] = `
-<PanelBody
+<[object Object]
```

Comment:
I wonder why the displayName we set is not being used here.

Reply:
Probably an enzyme bug :)